CHANGELOG-1.16.md

```diff
@@ -114,7 +114,7 @@ filename | sha512 hash
     * A sample cloud provider configure is: '{"vmType": "vmss", "useInstanceMetadata": true}'.
 * Volumes specified in a pod but not used in it are no longer unnecessarily formatted, mounted and reported in `node.status.volumesInUse`. ([#81163](https://github.com/kubernetes/kubernetes/pull/81163), [@jsafrane](https://github.com/jsafrane))
 * kubeadm: use etcd's /health endpoint for a HTTP liveness probe on localhost instead of having a custom health check using etcdctl ([#81385](https://github.com/kubernetes/kubernetes/pull/81385), [@neolit123](https://github.com/neolit123))
-* kubeamd: use the --pod-network-cidr flag to init or use the podSubnet field in the kubeadm config to pass a comma separated list of pod CIDRs.  ([#79033](https://github.com/kubernetes/kubernetes/pull/79033), [@Arvinderpal](https://github.com/Arvinderpal))
+* kubeadm: use the --pod-network-cidr flag to init or use the podSubnet field in the kubeadm config to pass a comma separated list of pod CIDRs.  ([#79033](https://github.com/kubernetes/kubernetes/pull/79033), [@Arvinderpal](https://github.com/Arvinderpal))
```

Comment:
@glefloch 
this changelog is autogenerated as you can see here:
https://github.com/kubernetes/kubernetes/commits/7e8f3f4ab87a683f4600383a5da0eec59551c46a/CHANGELOG-1.16.md
so we need to fix it in the PR description

i will ping the author of #79033
/close